STEERING-COMMITTEE.md

```diff
@@ -161,6 +158,17 @@ There are two ways to raise issues to the steering committee for decision:
    access.
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
+   
+   
+## Knative Trademark
+
+**Who owns the Knative Trademark?**
+
+Google owns the Knative word and logo trademarks. Given the open nature of Knative, users may freely use the Knative trademark to accurately refer to the project without prior written permission. Google looks to project leadership to keep an eye out for any misuses of the name. If people are concerned whether their use of the Knative mark is acceptable, we urge them to consult their own counsel or project leadership. 
+
+**Is IP assignment within the jurisdiction of Knative Steering?**
+
+No. Knative is an open source project governed under the terms of Google’s CLA. Contributing individuals and companies continue to own all copyrights and patents to their contributions. The CLA just provides Google and downstream users the ability to treat contributions as Apache 2.0-licensed along with the rest of the Knative project.
```

Comment:
According to https://github.com/knative/serving/blob/master/AUTHORS this statement appears false. When code is contributed to knative, the copyright header changes to "Knative Authors" which are apparently defined in this file https://github.com/knative/serving/blob/master/AUTHORS , control of this file is still unclear.